### PR TITLE
Implemented Task Display

### PR DESCRIPTION
**Pull request by faysal-alam-sazim**
### Description of Change
- Created card for tasks to display
- Added Edit, Delete, Mark as complete button
- Displayed sorted tasks based on priority
### Associated Ticket Link:
- https://trello.com/c/Q634gKUP
### Screenshots / Screen Recordings: 
Screen Shot 2024-07-11 at 12 12 29 AM

### CODE REVIEW

File: src/components/CreateTask/CreateTask.tsx
Comment (ishmam-iqbal-sazim) [inline src/components/CreateTask/CreateTask.tsx:44]: None of the fields contain validation. Please create a new ticket to address form validation.

File: package.json
Comment (ishmam-iqbal-sazim) [inline package.json:49]: If you are using mantine, do you need tailwind?

If not, please remove it from the dependencies.

File: postcss.config.js
Comment (ishmam-iqbal-sazim) [inline postcss.config.js:3]: Same comment as above.

File: src/localstorage/localstorage.ts
Comment (ishmam-iqbal-sazim) [inline src/localstorage/localstorage.ts:1]: Wasn't this already in the previous PR? It should not be visible here other than the changes. Point this branch to #3 rather than main

File: src/main.tsx
Comment (ishmam-iqbal-sazim) [inline src/main.tsx:6]: Should be a line break between library and relative imports

File: src/App.tsx
Comment (ishmam-iqbal-sazim) [inline src/App.tsx:3]: Relative imports should be grouped together.

File: src/Pages/HomePage/components/CreateTask/CreateTask.tsx
Comment (ishmam-iqbal-sazim) [inline src/Pages/HomePage/components/CreateTask/CreateTask.tsx:6]: Should be grouped with the library imports.

File: src/App.tsx
Comment (ishmam-iqbal-sazim) [inline src/App.tsx:14]:
```suggestion
  const [tasks, setTasks] = useState<Task[]>([]);
  
  useEffect(() => {
```

Add line breaks for better readability.

File: src/App.tsx
Comment (ishmam-iqbal-sazim) [inline src/App.tsx:24]:
```suggestion
  }, []);
  
  return (
```

Same comment as above.

File: src/Pages/HomePage/components/CreateTask/CreateTask.tsx
Comment (ishmam-iqbal-sazim) [inline src/Pages/HomePage/components/CreateTask/CreateTask.tsx:33]: Add line breaks for better readability.

File: src/components/CreateTask/CreateTask.tsx
Comment (faysal-alam-sazim) [reply src/components/CreateTask/CreateTask.tsx]: Created Ticket for form validation. link: https://trello.com/c/YFdHtcuq

File: package.json
Comment (faysal-alam-sazim) [reply package.json:49]: Bhai I've created a ticket for UI update. I will remove it then. I'm actually using tailwindcss for manual styling of some elements.

File: src/localstorage/localstorage.ts
Comment (faysal-alam-sazim) [reply src/localstorage/localstorage.ts:1]: I'm not sure if I can point this branch to previous branch now.

File: package.json
Comment (ishmam-iqbal-sazim) [reply package.json:49]: Okay no need to remove it then. Should be fine for this application.

But for future reference, it's better to stick to one ui library for a project especially considering mantine and tailwind follow different css paradigms (styled components and inline styling respectively).

File: src/localstorage/localstorage.ts
Comment (ishmam-iqbal-sazim) [reply src/localstorage/localstorage.ts:1]: If it is merged, point it to main.

Comment (faysal-alam-sazim) [reply src/localstorage/localstorage.ts:1]: Previous branch is merged with main

**Review by ishmam-iqbal-sazim — APPROVED** [review]
Approved pending an import fix comment for App.tsx. Address those changes and please rebase off main before merging.

**Pull request merged**
